test/core.spec.ts

```diff
@@ -896,6 +896,130 @@ contract("Core", (accounts) => {
     });
   });
 
+  describe("#redeem", async () => {
+    let subjectCaller: Address;
+    let subjectQuantityToRedeem: BigNumber;
+    let subjectSetToRedeem: Address;
+
+    const naturalUnit: BigNumber = ether(2);
+    let components: StandardTokenMockContract[] = [];
+    let componentUnits: BigNumber[];
+    let setToken: SetTokenContract;
+
+    beforeEach(async () => {
+      await deployCoreAndInitializeDependencies();
+
+      components = await coreWrapper.deployTokensAsync(2, ownerAccount);
+      await coreWrapper.approveTransfersAsync(components, transferProxy.address);
+
+      const componentAddresses = _.map(components, (token) => token.address);
+      componentUnits = _.map(components, () => ether(4)); // Multiple of naturalUnit
```

Comment:
The component units dont have to be a multiple of naturalUnit. Only the input quantity does

Reply:
I did this in my test for issue so that I could test what happens if the unit is not a multiple of the natural unit (3)